CMakeLists.txt

```diff
@@ -10,8 +10,3 @@ add_subdirectory(ext)
 add_subdirectory(lib)
 add_subdirectory(subsys)
 add_subdirectory(drivers)
-
```

Comment:
I believe we need

set(NRF_DIR ${CMAKE_CURRENT_LIST_DIR})

at the top of this file now, it used to be set (sometimes) from here:

https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/106/files#diff-af3b638bc2a3e6c650974192a53c7291L568

Reply:
There is a line setting that in boilerplate.cmake: https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/cmake/boilerplate.cmake#L26

Reply:
Yes, but AFAICT that file is not always executed. So it should be removed from there and added here.

Reply:
@SebastianBoe understood, I will do that.

Reply:
Nope, had to revert back because that makes the BOARD_ROOT not be taken into account